From 214d1615c480a29b9522181d1dc41ac8022a4492 Mon Sep 17 00:00:00 2001
From: Bala M T <mtbala@google.com>
Date: Wed, 18 Mar 2026 09:40:02 +0800
Subject: [PATCH 06/63] ANDROID: ABI: Update pixel symbol list

Adding the following symbols:
  - __traceiter_android_vh_pci_pm_verify_state
  - __tracepoint_android_vh_pci_pm_verify_state

Bug: 456059538
Bug: 493839773
Change-Id: Ic2e971c06092ed8d0fdcf9db6a21e5d3f1d078ea
Signed-off-by: Bala M T <mtbala@google.com>
(cherry picked from commit ef21d8d3de2777bd2c46ead723a127bc169782d8)
---
 gki/aarch64/abi.stg       | 27 +++++++++++++++++++++++++++
 gki/aarch64/symbols/pixel |  2 ++
 2 files changed, 29 insertions(+)

diff --git a/gki/aarch64/abi.stg b/gki/aarch64/abi.stg
index b25f951f38b4..94bd668fc175 100644
--- a/gki/aarch64/abi.stg
+++ b/gki/aarch64/abi.stg
@@ -420600,6 +420600,13 @@ function {
   parameter_id: 0x1b8590a8
   parameter_id: 0x20007040
 }
+function {
+  id: 0x9be19d09
+  return_type_id: 0x6720d32f
+  parameter_id: 0x18bd6530
+  parameter_id: 0x13580d6c
+  parameter_id: 0x0d1d7def
+}
 function {
   id: 0x9be1ca34
   return_type_id: 0x6720d32f
@@ -443976,6 +443983,15 @@ elf_symbol {
   type_id: 0x9bb62df9
   full_name: "__traceiter_android_vh_pagetypeinfo_show"
 }
+elf_symbol {
+  id: 0x5f55b542
+  name: "__traceiter_android_vh_pci_pm_verify_state"
+  is_defined: true
+  symbol_type: FUNCTION
+  crc: 0x5ef17f15
+  type_id: 0x9be19d09
+  full_name: "__traceiter_android_vh_pci_pm_verify_state"
+}
 elf_symbol {
   id: 0x13b0736e
   name: "__traceiter_android_vh_percpu_rwsem_down_read"
@@ -451797,6 +451813,15 @@ elf_symbol {
   type_id: 0x18ccbd2c
   full_name: "__tracepoint_android_vh_pagetypeinfo_show"
 }
+elf_symbol {
+  id: 0x01285174
+  name: "__tracepoint_android_vh_pci_pm_verify_state"
+  is_defined: true
+  symbol_type: OBJECT
+  crc: 0x7c5aa8a7
+  type_id: 0x18ccbd2c
+  full_name: "__tracepoint_android_vh_pci_pm_verify_state"
+}
 elf_symbol {
   id: 0xa4c454d8
   name: "__tracepoint_android_vh_percpu_rwsem_down_read"
@@ -526886,6 +526911,7 @@ interface {
   symbol_id: 0x3246acbb
   symbol_id: 0xb4d5ffdc
   symbol_id: 0x3c5e668a
+  symbol_id: 0x5f55b542
   symbol_id: 0x13b0736e
   symbol_id: 0x12b996bc
   symbol_id: 0xc72f2012
@@ -527755,6 +527781,7 @@ interface {
   symbol_id: 0x4a5e6e41
   symbol_id: 0x352038ba
   symbol_id: 0x2f61d5a4
+  symbol_id: 0x01285174
   symbol_id: 0xa4c454d8
   symbol_id: 0xffa6607a
   symbol_id: 0x7d42b7c8
diff --git a/gki/aarch64/symbols/pixel b/gki/aarch64/symbols/pixel
index 7d47a64bc75e..ccf9b04e589d 100644
--- a/gki/aarch64/symbols/pixel
+++ b/gki/aarch64/symbols/pixel
@@ -2831,6 +2831,7 @@
   __traceiter_android_vh_mm_kcompactd_cpu_online
   __traceiter_android_vh_page_cache_ra_order_bypass
   __traceiter_android_vh_page_cache_readahead_start
+  __traceiter_android_vh_pci_pm_verify_state
   __traceiter_android_vh_platform_pci_choose_state
   __traceiter_android_vh_platform_pci_get_power_state
   __traceiter_android_vh_platform_pci_power_manageable
@@ -2985,6 +2986,7 @@
   __tracepoint_android_vh_mm_kcompactd_cpu_online
   __tracepoint_android_vh_page_cache_ra_order_bypass
   __tracepoint_android_vh_page_cache_readahead_start
+  __tracepoint_android_vh_pci_pm_verify_state
   __tracepoint_android_vh_platform_pci_choose_state
   __tracepoint_android_vh_platform_pci_get_power_state
   __tracepoint_android_vh_platform_pci_power_manageable
-- 
2.50.1

